### ISSUE TITLE
Missing language of page

### ISSUE THREAD

**Issue by lizgarciao** [pulibrary/audit-Liz#6]
### URL
https://lizgarciao.github.io/

### Issues
Missing language of page

### Severity
Critical

### WCAG Conformance level (A, AA, AAA)
A

### User impact
[Understanding Success Criterion 3.1.1: Language of Page | WAI | W3C](https://www.w3.org/WAI/WCAG21/Understanding/language-of-page.html#:~:text=AA%20Success%20Criterion.-,Benefits,-This%20Success%20Criterion)

### Recommended fix
Include lang="en" in html tag

### Acceptance criteria
- [x] I can successfully use this feature with Voice Over in Safari.
- [x] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
